Card/src/main/java/com/paypal/android/card/ConfirmPaymentSourceResult.kt

```diff
@@ -0,0 +1,9 @@
+package com.paypal.android.card
+
+import com.paypal.android.core.OrderData
+import com.paypal.android.core.OrderError
+
+data class ConfirmPaymentSourceResult(
+    val response: OrderData? = null,
+    val error: OrderError? = null
+)
```

Comment:
We should align across iOS & Android how we name these results & how they're structured. I was thinking of the Core module having something like

`PaymentResult` (has the standard order & error) and then subclasses in each module like `CardResult`, `VenmoResult` etc that can extend each and provide additional params.

I know @minhthenguyen had another idea. We should definitely keep this naming and structure in parity across iOS and Android.

Reply:
Yeah I'm down for unifying the structure. The response objects do seem pretty uniform I think this will help to improve the developer experience.